calculator.js

```diff
@@ -1,60 +1,26 @@
-// --CALCULATOR MATH FUNCTIONS--
-const add = ((num1, num2) => {
-    return num1 + num2;
-});
+//--ALL MATH FUNCTIONS--
+const MATH_FUNCS = {
+    add: (num1, num2) => num1 + num2,
+    subtract: (num1, num2) => num1 - num2,
+    multiply: (num1, num2) => num1 * num2,
+    divide: (num1, num2) => num1 / num2
+}
 
-const subtract = ((num1, num2) => {
-    return num1 - num2;
-});
 
-const multiply = ((num1, num2) => {
-    return num1 * num2;
-});
+const calculateDisplay = () => {
 
-const divide = ((num1, num2) => {
-    return num1 / num2;
-});
+/*
 
-// OPERATE FUNCTION WITH SWITCH STATEMENT
-/* 
-Wow! Eric tells me this is a great way to write this function. 
-It is so short and simple and nice. 
-Way to go, me (Ellen Jaquette).
-*/
-const operate = (operator, num1, num2) => {
-switch(operator) {
-    case '+':
-        return add(num1, num2);
-      break;
-    case '-':
-        return subtract(num1, num2);
-      break;
-    case 'x':
-        return multiply(num1, num2);
-      break;
-    case '/':
-        return divide(num1, num2);
-    break
-    default:
-       return 'ERROR';
-  } 
-}
-// OPERATE COMMAND WITH IF/ELSE STATEMENT
-// const operate = ((operator, num1, num2) => {
-//     if (operator == '+') {
-//         return add(num1, num2);
-//     } else if (operator == '-') {
-//         return subtract(num1, num2);
-//     } else if (operator == 'x') {
-//         return multiply(num1, num2);
-//     } else(operator == '/')
-//     return divide(num1, num2);
-
-// })
+ But one thing you'll also want to do is make it so that, if the "key" doesn't
+ exist in your lookup table, then you will want to set the display to
+ 'ERROR` here. 
 
-const calculateDisplay = () => {
+ Remember, you want to find the value of the appropriate key in the lookup table;
+ it will return a function.
+ You then want to run that function. You can do it all in one line (or two).
+ */
 
-    const display = operate(operator, num1, num2);   
+    const display = MATH_FUNCS[operator](num1, num2);
     displayElem.innerText = display;
```

Comment:
OK, we're ALMOST done with this round. 

In a later segment of code you check that `num1` and `num2` both exist. Otherwise you set the `displayElem` innerText to `ERROR`. 

I think you should do the same thing here, but for the operator. This is essentially checking to see if `MATH_FUNCs` has a value for the key specified by `operator`. 

#### Example
 if [somehow] a operator came to this `calculateDisplay` function as `exp`, and you didn't have that in your `MATH_FUNCS` lookup table, then what would happen?

What should happen is that the text in the display should display `ERROR`. 

Advanced: See if you can do this without using an if/else!